### PR TITLE
Remove obsolete files

### PR DESCRIPTION
**Pull request by danielhollas**
As it says on the tin --- none of these files seem to be used anymore.

### CODE REVIEW

**Comment by unkcpz** [comment]
I agree. But no idea what those files were for, better to have @yakutovicha double check on it.

File: __init__.py
Comment (yakutovicha) [inline __init__.py:1]: This one was used in the infancy of AiiDAlab to make an app importable. Not needed anymore.

File: install.sh
Comment (yakutovicha) [inline install.sh:1]: This is from the old days when the jupyter extensions had to be activated after an install. In this particular case, we don't even use the external `fileupload` widget anymore

File: miscellaneous/images/computer.png
Comment (yakutovicha) [inline miscellaneous/images/computer.png:1]: The computer setup widget that used this image was removed [here](https://github.com/aiidalab/aiidalab-widgets-base/pull/344/files#diff-41129b0f1db994ae59e278b44419d6edd35ba16df9ae0b4123634e40ba1de293).

File: eln_import.ipynb
Comment (yakutovicha) [inline eln_import.ipynb:1]: An old link to keep backwards compatibility.

**Review by yakutovicha — APPROVED** [review]
Added some history in the comments

**Pull request merged**